.github/workflows/scripts/css_check.py

```
@@ -0,0 +1,350 @@
+        if os.path.isdir(path):
+            validated_dirs.append(path)
+        elif os.path.isfile(path):
+            validated_dirs.append(os.path.dirname(path))
+        else:
+            raise ValueError(
+                f"Invalid path: {path}. Must be an existing file or directory."
+            )
+    return validated_dirs
+
+
+def main():
```

Ruff (0.12.2)

`main` is too complex (11 > 10)

(C901)